unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/EmailPromptHUD/EmailPromptHUDController.cs

```diff
@@ -1,34 +1,144 @@
 ﻿using UnityEngine;
-using UnityEngine.UI;
-using TMPro;
 using DCL.Interface;
+using DCL.Helpers;
+using System;
+using System.Collections;
+using DCL.Tutorial;
 
-public class EmailPromptHUDController : MonoBehaviour
+public class EmailPromptHUDController : IHUD
 {
-    public TMP_InputField inputField;
-    public GameObject cryptoUserMessage;
-    public GameObject nonCryptoUserMessage;
-    public Button sendButton;
+    const float POPUP_DELAY = 60;
 
-    void Awake()
+    EmailPromptHUDView view;
+
+    bool isRendererEnabled = false;
```

Comment:
Why we are using this bool instead of querying the renderer state `ScriptableObject`?